scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+script: scpSHI_Jetz_complete.R
+description: 
+  This script loads the data needed to calculate the area and connectivity scores 
+  to calculate the Species Habitat Index (SHI) using Global Forest Watch data to measure 
+  changes in the habitat of the species
+inputs:
+  species:
+    label: species
+    description: Scientific name of the species.
+    type: text
+    example: Saguinus oedipus
+  srs:
+    label: spatial reference system
+    description: Reference system for the area of interest. It can be 1) the Spatial Reference System Identifier (SRID), 2) the authority name (e.g. EPSG) with the SRID or 3) the description of the spatial reference system details (e.g. [https://spatialreference.org/]). If just the SRID is given and the software can not find the reference system try options 2 or 3.
+    type: text
+    example: EPSG:3116
+  elev_buffer:
+    label: elevation buffer
+    description: Elevation buffer in meters to add (or substract) to the reported species elevation range. Defalut is zero. Positive values will increase the range in that magnitude and negative values will reduce the range in that magnitude.
+    type: int
+    example: 0
+  forest_threshold:
+    label: forest threshold
+    description: 
+      Percentage of tree cover required by the species, that will be used to filter the Global Forest Watch data. 
+      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
```

Comment:
Just a little rephrase: "Threshold for required percentage of tree cover based on suitable habitat of the species. Acts as a filter for the Global Forest Watch Data. If not available, use Map of Life Values..."